array_api_compat/common/_helpers.py

```diff
@@ -166,6 +171,35 @@ def is_jax_array(x):
 
     return isinstance(x, jax.Array) or _is_jax_zero_gradient_array(x)
 
+
+def is_sparse_array(x) -> bool:
```

Comment:
Is the package still called pydata/sparse? I worry this function name could be confusing out of context.

Reply:
I've taken to just calling it `sparse` in the past year to 18 months -- PyData/Sparse is a mouthful to say and difficult to type right.

Reply:
I've reworded the docstring in https://github.com/data-apis/array-api-compat/pull/134/commits/fb14cc815b9cb0e4b7d2ae8180b685537443c827 for clarity, and marking as resolved -- however, please mark as un-resolved and propose a better name if you can think of one.

Reply:
I'm curious what other people think of this. @rgommers

My worry is that `is_sparse_array` will be a confusing name, since it is not checking ia an array is sparse, but rather if it comes from the *library* "sparse". But this also matches the convention of the other functions in this namespace.

Reply:
Done in https://github.com/data-apis/array-api-compat/pull/134/commits/7ebc3c096ad022f1b3448ca233377a3ec72b3af8.